pkg/cmd/cluster/config_resource.go

```diff
@@ -271,3 +341,72 @@ func (w *configObjectsWrapper) genConfigSpecs(objects *ConfigRelatedObjects, com
 	}
 	return ret, nil
 }
+
+func (w *configObjectsWrapper) genConfigSpecsByCompDef(comp *appsv1alpha1.Component, compDefs []*appsv1alpha1.ComponentDefinition) ([]*configSpecMeta, error) {
+	var (
+		ret []*configSpecMeta
+	)
+	targetCompDef := comp.Labels[constant.ComponentDefinitionLabelKey]
```

Comment:
The Component CR object does not have this label; the CompDef associated with the Component is in the Component.Spec.CompDef field.